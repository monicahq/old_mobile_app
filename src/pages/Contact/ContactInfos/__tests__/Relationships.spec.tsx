import {contact1} from '@src/__mock__/contact';
import {shallow} from 'enzyme';
import toJson from 'enzyme-to-json';
import React from 'react';
import {Relationships} from '../Relationships';

describe('Pages', () => {
  describe('Contact / Relationships', () => {
    it('should renders correctly', () => {
      expect(
        toJson(shallow(<Relationships contact={contact1} />))
      ).toMatchSnapshot();
    });
  });
});
