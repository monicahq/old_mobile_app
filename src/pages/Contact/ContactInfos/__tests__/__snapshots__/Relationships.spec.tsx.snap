// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pages Contact / Relationships should renders correctly 1`] = `
<ForwardRef(View)
  style={
    Object {
      "marginTop": 5,
    }
  }
>
  <ForwardRef(Text)
    style={
      Object {
        "marginVertical": 30,
        "textAlign": "center",
      }
    }
  >
    No relationship
  </ForwardRef(Text)>
</ForwardRef(View)>
`;
